### ISSUE TITLE
data leak risk

### ISSUE THREAD

**Issue by DopamineLcy** [whaleloops/KEPT#4]

**Comment by whaleloops** [comment]
Hi, thanks for your question.

Yes, data leakage might occur. One could try fine-tune on MIMIC-IV and see the effect of data leakage. But the pretaining task (predict token masking) is very different from icd coding task (multi label classification), which is why pretrained Clinical-Longformer (_w/o HSAP &Prompt_ row in [table 1](https://arxiv.org/pdf/2210.03304.pdf)) does not outperform unpretrained but task-specific engineered model such as JointLAAT. 

p.s. We also explored pretraining on other private corpus with phrase and sentence masking and then fientuned on MIMIC-III. So no leakage in this paper. Results are shown [here](https://arxiv.org/abs/2211.13813).
